### PR TITLE
Add 1 to number represented as array of digits

### PR DESCRIPTION
**Pull request by HariniJeyaraman**
**Fixes issue:**

#4977

**Changes:**

Add 1 to a number represented as an array of digits in C++

### CODE REVIEW

**Comment by HariniJeyaraman** [comment]
@arnavb I have implemented it in C++. Please review my code

**Comment by HariniJeyaraman** [comment]
@arnavb Please review my PR

File: code/unclassified/src/add_one_to_number/add_one_to_number.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        while (1) {
+            if (A[ind] + A[ind + 1] == 0) {
+                c++;
+                ind++;
+            } else
+                break;
+        }
+    }
+    // cout<<" C is : " <<c<<"\n";
+    for (int i = A.size() - 1; i >= 0; i--) {
```
Comment (arnavb) [inline code/unclassified/src/add_one_to_number/add_one_to_number.cpp:33]: Pre-increment in all loops.

Comment (HariniJeyaraman) [reply code/unclassified/src/add_one_to_number/add_one_to_number.cpp]: done

**Pull request merged**